boards/arm/rddrone_fmuk66/rddrone_fmuk66.dts

```diff
@@ -106,6 +109,18 @@
 		status = "okay";
 	};
 
+	servo: servo {
+		compatible = "pwm-servo";
```

Comment:
where is this compatible found?

Reply:
Servo motor is in the form of an app rather than a driver.
Please check out this README -
https://github.com/zephyrproject-rtos/zephyr/blob/7ad2e604bb3155c09a484717e1252c8f404640c0/samples/basic/servo_motor/README.rst

Reply:
this compatible is not available outside of that sample.

Reply:
@gmarull  Just to clarify.. Are you suggesting us to keep this servo node in the board's overlay file rather than in board dts ?
Like what's present for bbc_microbit in samples/basic/servo_motor/boards/bbc_microbit.overlay

Reply:
I guess in your case you have a board that really has servo specific connectors, unlike bcc_microbit which is a general purpose board. So, it likely makes sense to move servo-motor compatible to `dts/bindings/...` folder if DT maintainers are fine with it.

Reply:
@gmarull 
Only adding yaml binding for pwm-servo will be sufficient ? Or will you still need a driver like - drivers/led/led_pwm.c

Reply:
bindings are sufficient, they are really the schema for DT content. It is not necessary to provide a driver for each DT compatible, in fact, some may never have one (e.g. the CPU). How DT hardware info is used may vary.

Reply:
@gmarull .. Kindly check again and let us know if you are satisfied.

Reply:
@gmarull .. The current servo_motor sample directory already has the pwm-servo.yaml present in -samples/basic/servo_motor/dts/bindings/pwm-servo.yaml
Now, If I add your proposed changes in the generic dts/bindings directory then servo_motor sample build fails. 

So, I think with due respect to the current servo_motor software design, we should add our board changes in the overlay file - *zephyr/samples/basic/servo_motor/boards/rddrone_fmuk66.overlay*

Reply:
Yes, that is also a valid option, definitely.

Reply:
@gmarull.. kindly re-review